core/domain/recommendations_services_test.py

```diff
@@ -254,7 +255,7 @@ def setUp(self):
 
         self.admin_id = self.get_user_id_from_email(self.ADMIN_EMAIL)
         user_services.get_or_create_user(
-            self.admin_id, self.ADMIN_EMAIL)
+            self.admin_id, self.ADMIN_EMAIL, feconf.ROLE_EXPLORATION_EDITOR)
```

Comment:
In following lines set_admins function is used which will anyway update its role to admin. 

So, I looked at it as "a normal user registers here and then its role is updated to admin by set_admins".

But I'll update it for clarity if you say so.

updated.